docs/frontend/nx-monorepo.md

```diff
@@ -75,6 +75,29 @@ A set of helm templates have been created in `{repo}/helm/cas-registration/templ
 
 ## Nx Generators 🦾
 
+### Creating new projects 🏗
+
+For further information, see the [Nx docs about @nx/next generators](https://nx.dev/nx-api/next/generators/application).
+
+#### Eslint
+
+When creating a new project with Nx, there is some setup that needs to be done to ensure that eslint works correctly with typescript. Nx doesn't set this up by default since there is a performance hit though we are using typescript in all of our projects. More information about this can be found in the [Nx docs](https://nx.dev/latest/react/guides/eslint).
+
+This is done by adding the following to the `eslintrc.json` file in the project you created in `apps/{project}/.eslintrc.json` or `libs/shared/{project}/.eslintrc.json`:
```

Comment:
I'm still not sure I follow this (though I think that has more to do with not having more general Nx docs than these docs specifically). Is the following correct?
```suggestion
When creating a new project or library with Nx, there is some setup that needs to be done to ensure that eslint works correctly with typescript. Nx creates an eslint config file by default when creating when new projects or libraries, but Nx doesn't include typescript configuration by default (there's a performance hit). Since we use typescript in all our projects, we need to add configuration for it. More information about this can be found in the [Nx docs](https://nx.dev/latest/react/guides/eslint).

This is done by adding the following to the autogenerated `eslintrc.json` file (`apps/{project}/.eslintrc.json` if you made a new project or `libs/shared/{project}/.eslintrc.json if you made a new library)`:
```

Reply:
Thanks, this makes sense even if every library is a project in nx (each have their own `project.json`. Though I think this is a good suggestion and makes it easier to understand! 🎉